Modules/_collectionsmodule.c

```diff
@@ -1510,26 +1627,24 @@ deque_richcompare(PyObject *v, PyObject *w, int op)
     return NULL;
 }
 
+/*[clinic input]
+@text_signature "([iterable[, maxlen]])"
+_collections.deque.__init__ as deque___init__
```

Comment:
Sorry for the PR churn; my previous suggestion should have been:

```suggestion
_collections.deque.__init__ as deque_init
```